pkg/agent/agent.go

```diff
@@ -100,6 +100,20 @@ func UpgradeRelease(actionConfig *action.Configuration, name, valuesYaml string,
 	return res, nil
 }
 
+func RollbackRelease(actionConfig *action.Configuration, releaseName string, revision int) (*release.Release, error) {
+	log.Printf("Rolling back %s to revision %d.", releaseName, revision)
+	rollback := action.NewRollback(actionConfig)
+	rollback.Version = revision
+	err := rollback.Run(releaseName)
+	if err != nil {
+		return nil, err
+	}
+
+	// The Helm3 rollback action does not return the new release, unlike the helm2 equivalent,
```

Comment:
Maybe like this (if we care about detail)?
```suggestion
	// The Helm 3 rollback action does not return the new release, unlike the Helm 2 equivalent,
```